nsqproducer/producer.go

```diff
@@ -16,6 +16,7 @@ import (
 type Producer interface {
 	Publish(ctx context.Context, topic string, message NsqMessageSerialize) error
 	DeferredPublish(ctx context.Context, topic string, delay int64, message NsqMessageSerialize) error
+	Stop()
```

Comment:
Shouldn't you implement this method in the `NsqProducer` (non-LB)?

Reply:
It's already there.